scripts/generateLumoAutoCompleteCss.js

```diff
@@ -0,0 +1,83 @@
+const fs = require('fs');
+const path = require('path');
+
+const PACKAGE_BASE = 'packages/vaadin-lumo-styles';
+
+const styleModules = ['color', 'sizing', 'spacing', 'style', 'typography', 'user-colors'];
+const utilityModules = [
```

Comment:
Should we instead check all the file names in `utilities` folder and get modules from there?
This might be useful if at some point we decide to add more files / remove some of them.

Not a blocker, can be also considered later (as we don't have any plans for utility classes).

Reply:
Sure, done.